app/cleanup.py

```diff
@@ -26,6 +26,11 @@ def process(message: Message, tx_id: TX_ID):
     # all artifacts require removing from outputs bucket
     sdx_app.gcs_delete(file, CONFIG.OUTPUT_BUCKET_NAME)
 
+    # The original filename doesn't have this prefix, so remove it in order to delete it
+    if file_name.startswith('739-') or file_name.startswith('738-'):
+        file_name = file_name[4:]
+        logger.info(f"Renamed feedback filename to delete from input bucket: {file_name}")
```

Comment:
This might be a bit misleading as you're not actually renaming the filename. Just setting the filename for cleanup